content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+
+Cada equipo está organizado alrededor de un lenguaje común. Cada comunidad puede luego producir múltiples localizaciones recurso de varios proyectos.
+
+Sus responsabilidades son:
+* Traducciones de cualquiera de los materiales io.js que ellos crean relevantes para su comunidad.
+* Revisar procesos teniendo traducciones actualizadas y con alta calidad.
+* Redes sociales en su lenguaje.
+* Promoción de conferenciantes de io.js para reuniones y conferencias en su lenguaje.
+
+Cada comunidad de lenguaje mantiene sus afiliados.
+
+* [iojs-ar - Árabe (اللغة العربية)](http://github.com/iojs/iojs-ar)
+* [iojs-bg - Bulgara (български език)](http://github.com/iojs/iojs-bg)
+* [iojs-bn - Bengalí (বাংলা)](http://github.com/iojs/iojs-bn)
+* [iojs-cn - China (中文)](http://github.com/iojs/iojs-cn)
+* [iojs-cs - Checa (Český Jazyk)](http://github.com/iojs/iojs-cs)
+* [iojs-da - Danesa (Dansk)](http://github.com/iojs/iojs-da)
+* [iojs-de - Alemana (Deutsch)](http://github.com/iojs/iojs-de)
+* [iojs-el - Griega (Ελληνικά)](http://github.com/iojs/iojs-el)
+* [iojs-es - Española (Español)](http://github.com/iojs/iojs-es)
+* [iojs-fa - Persa (فارسی)](http://github.com/iojs/iojs-fa)
+* [iojs-fi - Finesa (Suomi)](http://github.com/iojs/iojs-fi)
+* [iojs-fr - Francesa (Français)](http://github.com/iojs/iojs-fr)
+* [iojs-he - Hebrea (עברית)](http://github.com/iojs/iojs-he)
+* [iojs-hi - Hindi (फिजी बात)](http://github.com/iojs/iojs-hi)
+* [iojs-hu - Húngara (Magyar)](http://github.com/iojs/iojs-hu)
+* [iojs-id - Indonesia (Bahasa Indonesia)](http://github.com/iojs/iojs-id)
+* [iojs-it - Italiana (Italiano)](http://github.com/iojs/iojs-it)
+* [iojs-ja - Japonesa (日本語)](http://github.com/iojs/iojs-ja)
+* [iojs-ka - Georgiana (ქართული)](http://github.com/iojs/iojs-ka)
+* [iojs-ko - Coreana (조선말)](http://github.com/iojs/iojs-ko)
+* [iojs-mk - Macedonia (Mакедонски)](http://github.com/iojs/iojs-mk)
+* [iojs-ms - Malaya (بهاس ملايو)](http://github.com/iojs/iojs-ms)
+* [iojs-nl - Holandesa (Nederlands)](http://github.com/iojs/iojs-nl)
+* [iojs-no - Noruega (Norsk)](http://github.com/iojs/iojs-no)
+* [iojs-pl - Polish (Język Polski)](http://github.com/iojs/iojs-pl)
+* [iojs-pt - Portuguesa (Português)](http://github.com/iojs/iojs-pt)
+* [iojs-ro - Rumana (Română)](http://github.com/iojs/iojs-ro)
+* [iojs-ru - Rusa (Русский)](http://github.com/iojs/iojs-ru)
+* [iojs-sv - Sueca (Svenska)](http://github.com/iojs/iojs-sv)
+* [iojs-ta - Tamil (தமிழ்)](http://github.com/iojs/iojs-ta)
+* [iojs-tr - Turca (Türkçe)](http://github.com/iojs/iojs-tr)
+* [iojs-tw - Taiwanesa (Hō-ló)](http://github.com/iojs/iojs-tw)
+* [iojs-uk - Ucraniana (Українська)](http://github.com/iojs/iojs-uk)
+* [iojs-vi - Vietnamita (Tiếng Việtnam)](http://github.com/iojs/iojs-vi)
+
+
+### [Evangelism](https://github.com/iojs/evangelism)
+
+El grupo de trabajo evangelism promueve los logros de io.js y hace saber a la comunidad como pueden involucrarse.
```

Comment:
Same as the previous. Keeping the first occurrence of an acronym with its meaning.